frontend/sozisel-app/src/components/Login/Login.tsx

```diff
@@ -0,0 +1,85 @@
+import "./login.scss";
+
+import * as Yup from "yup";
+
+import Button from "../utils/Button/Button";
+import Card from "../utils/Card/Card";
+import ErrorMessage from "../utils/Input/ErrorMessage";
+import Input from "../utils/Input/Input";
+import Navbar from "../Navbar/Navbar";
+import { ReactElement } from "react";
+import conference_img from "../../assets/conference_img.png";
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router";
+import { useLoginMutation } from "../../graphql";
+import { yupResolver } from "@hookform/resolvers/yup";
+
+const fieldRequiredError = "To pole jest wymagane!";
```

Comment:
To be honest, I know that we do not intend to use internationalization but keeping json files with translations will make everything look waaay cleaner instead of using hard coded strings in code. Use `react-i18next`.